calendar/quickstart/quickstart.py

```diff
@@ -33,7 +33,7 @@
 service = build('calendar', 'v3', http=creds.authorize(Http()))
 
 # Call the Calendar API
-now = datetime.datetime.utcnow().isoformat() + 'Z' # 'Z' indicates UTC time
+now = datetime.datetime.utcnow().isoformat() + 'Z'  # 'Z' indicates UTC time
```

Comment:
I have only added it because my linter recommends to add two spaces before an inline comment (PEP 8) - "You should use two spaces after a sentence-ending period in multi- sentence comments, except after the final sentence." https://www.python.org/dev/peps/pep-0008/?

But I've removed it almost mechanically, I don't think such small things matter too much